RetsTalk/RetsTalk/Retrospect/Controller/RetrospectListViewController.swift

```diff
@@ -33,46 +47,120 @@ final class RetrospectListViewController: UIViewController {
     override func viewDidLoad() {
         super.viewDidLoad()
         
-        setUpNavigationBar()
         retrospectListView.setTableViewDelegate(self)
+        observeRetrospects()
+        addCreateButtonTapAction()
+        
+        Task {
+            await retrospectManager.fetchRetrospects(of: [.pinned, .inProgress, .finished])
+            sortAndSendRetrospects()
+        }
     }
     
-    // MARK: Custom method
+    // MARK: RetsTalk lifecycle method
     
-    private func setUpNavigationBar() {
-        title = Texts.titleLabelText
+    override func setupNavigationBar() {
+        title = Texts.navigationTitle
         navigationController?.navigationBar.prefersLargeTitles = true
+        
+        let settingsButton = UIBarButtonItem(
+            image: UIImage(systemName: Texts.settingButtonImageName),
+            style: .plain,
+            target: self,
+            action: #selector(didTapSettings)
+        )
+        
+        navigationController?.navigationBar.prefersLargeTitles = true
+        navigationItem.rightBarButtonItem = settingsButton
+        navigationItem.rightBarButtonItem?.tintColor = .black
+    }
+    
+    // MARK: Custom method
+    
+    private func sortAndSendRetrospects() {
+        Task {
+            let sortedRetrospects = RetrospectSortingHelper.execute(await retrospectManager.retrospects)
+            retrospectsSubject.send(sortedRetrospects)
+        }
+    }
+    
+    private func observeRetrospects() {
+        retrospectsSubject
+            .receive(on: RunLoop.main)
+            .sink { [weak self] _ in
+                guard let self = self else { return }
+                retrospectListView.reloadData()
+            }
+            .store(in: &subscriptionSet)
+    }
+    
+    @objc func didTapSettings() {
```

Comment:
외부에서 호출할 일이 없다면 접근 제어자 설정을 해주는게 좋겠네요!
```suggestion
    @objc private func didTapSettings() {
```